.github/workflows/tests.yml

```diff
@@ -36,32 +32,44 @@ jobs:
       matrix:
         os: [ubuntu-latest, windows-latest]
         python-version: ['3.9','3.10','3.11']
+        include:
+          - os: windows-latest
+            # this mamba path will be interpreted by bash instead of the file system
+            # hence the double \\ are necessary
+            mamba-path: C:\\Users\\runneradmin\\miniconda3\\condabin\\mamba
+            miniconda-path: C:\Users\runneradmin\miniconda3
+            pycache-path: C:\Users\runneradmin\pycache
```

Comment:
because it's the only one that will be interpreted by bash which will interpret eg \U as an unknown control character hence the double \ are necessary. The other paths are only evaluated by the windows file system and therefore don't need it.